drivers/flash/flash_stm32_ospi.c

```diff
@@ -0,0 +1,1676 @@
+/*
+ * Copyright (c) 2022 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT st_stm32_ospi_nor
+
+#include <errno.h>
+#include <kernel.h>
+#include <toolchain.h>
+#include <arch/common/ffs.h>
+#include <sys/util.h>
+#include <soc.h>
+#include <drivers/pinctrl.h>
+#include <drivers/clock_control/stm32_clock_control.h>
+#include <drivers/clock_control.h>
+#include <drivers/flash.h>
+#include <dt-bindings/flash_controller/ospi.h>
+
+#include "spi_nor.h"
+#include "jesd216.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(flash_stm32_ospi, CONFIG_FLASH_LOG_LEVEL);
+
+#define STM32_OSPI_RESET_GPIO DT_INST_NODE_HAS_PROP(0, reset_gpios)
+#if STM32_OSPI_RESET_GPIO
+#include <drivers/gpio.h>
+#endif /* STM32_OSPI_RESET_GPIO */
```

Comment:
Let's remove the conditional here and include in any case.
This is simpler and  how it is done in general.

Reply:
done